controller/create_or_update_jobs.py

```diff
@@ -33,6 +33,7 @@
 
 
 log = logging.getLogger(__name__)
+tracer = trace.get_tracer(__name__)
```

Comment:
Hah, actually using `__name__` is not a bad idea, but I didn't mean to stretch the analogy to loggers that far!